### PR TITLE
Feature/get way points

### PR DESCRIPTION
**Pull request by KhrystynaDubovyk**
A new folder GetWayPoints was created on guidelines.
Index with description, tables and diagram was created.
Item GetWayPoints was added to navigational pannel.

### CODE REVIEW

File: docs/Navigation/GetWayPoints/index.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+|||
+GetWayPoints
+![GetWayPoints](./assets/GetWayPoints.png)
+|||
+
+### JSON Messages Examples
+
+### Example Request
+
+```
```
Comment (ShobhitAd) [inline docs/Navigation/GetWayPoints/index.md]:
```suggestion
```json
```

File: docs/Navigation/GetWayPoints/index.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+	{
+		"wayPointType" : "ALL",
+		"appID" : 26743
+	}
+}
+```
+
+### Example Response
+
+```
```
Comment (ShobhitAd) [inline docs/Navigation/GetWayPoints/index.md]:
```suggestion
```json
```

File: docs/Navigation/GetWayPoints/index.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+			}
+		],
+		"appID" : 26743
+	}
+}
+```
+
+### Example Error
+
+```
```
Comment (ShobhitAd) [inline docs/Navigation/GetWayPoints/index.md]:
```suggestion
```json
```

File: docs/Navigation/OnWayPointChange/index.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+#### Parameters
+
+|Name|Type|Mandatory|Additional|Description|
+|:---|:---|:--------|:---------|:----------|
+|wayPoints|[Common.LocationDetails](../../common/structs/#locationdetails)|true|array: true<br>minsize: 1<br>maxsize: 10||
+
+### Sequence Diagrams
+|||
+OnWayPointChange
+![OnWayPointChange](./assets/OnWayPointChange.jpg)
```
Comment (ShobhitAd) [inline docs/Navigation/OnWayPointChange/index.md]:
```suggestion
![OnWayPointChange](./assets/OnWayPointChange.png)
```

File: docs/Navigation/SubscribeWayPoints/index.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+### Request
+
+!!! MUST
+Send future waypoint updates to SDL using [Navigation.OnWayPointChange](../onwaypointchange)
+!!!
+
+#### Parameters
+
+This RPC has no additional parameter requirements
```
Comment (ShobhitAd) [inline docs/Navigation/SubscribeWayPoints/index.md]:
```suggestion
There are no defined parameters for this function
```

File: docs/Navigation/SubscribeWayPoints/index.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+
+### Response
+
+#### Parameters
+
+This RPC has no additional parameter requirements
```
Comment (ShobhitAd) [inline docs/Navigation/SubscribeWayPoints/index.md]:
```suggestion
There are no defined parameters for this function
```

File: docs/Navigation/SubscribeWayPoints/index.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+
+### Sequence Diagrams
+|||
+SubscribeWayPoints
+![SubscribeWayPoints](./assets/SubscribeWayPoints.jpg)
```
Comment (ShobhitAd) [inline docs/Navigation/SubscribeWayPoints/index.md]:
```suggestion
![SubscribeWayPoints](./assets/SubscribeWayPoints.png)
```

File: docs/Navigation/UnsubscribeWayPoints/index.md [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+
+### Request
+
+!!! MUST
+Stop sending waypoint updates to SDL
+!!!
+
+#### Parameters
+
+This RPC has no additional parameter requirements
```
Comment (ShobhitAd) [inline docs/Navigation/UnsubscribeWayPoints/index.md]:
```suggestion
There are no defined parameters for this function
```

File: docs/Navigation/UnsubscribeWayPoints/index.md [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+
+### Response
+
+#### Parameters
+
+This RPC has no additional parameter requirements
```
Comment (ShobhitAd) [inline docs/Navigation/UnsubscribeWayPoints/index.md]:
```suggestion
There are no defined parameters for this function
```

File: docs/Navigation/UnsubscribeWayPoints/index.md [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+
+### Sequence Diagrams
+|||
+UnsubscribeWayPoints
+![UnsubscribeWayPoints](./assets/UnsubscribeWayPoints.jpg)
```
Comment (ShobhitAd) [inline docs/Navigation/UnsubscribeWayPoints/index.md]:
```suggestion
![UnsubscribeWayPoints](./assets/UnsubscribeWayPoints.png)
```

File: docs/Navigation/SubscribeWayPoints/index.md (hunk 5 above)
Comment (jacobkeeler) [reply docs/Navigation/SubscribeWayPoints/index.md]: This is the language used throughout the rest of the docs, so I'm inclined toward keeping it consistent. I also believe that it is clear enough as it is worded now.

Comment (ShobhitAd) [reply docs/Navigation/SubscribeWayPoints/index.md]: Additional parameter requirements makes it sound like there are some base parameters which are always required. But if it's the convention then I guess it works

**Pull request merged**